examples/scroll.py

```diff
@@ -28,7 +28,7 @@
 DIR_RIGHT = 4
 
 zoom_factor = 8
-
+ARROWCOLOR="black"
```

Comment:
Spaces between the = sign would be nice.

Reply:
Pygame uses black for formatting. I believe `setup.py lint` will run it over everything, but that's a very new workflow. You can `black` individual files or folders.